src/com/facebook/buck/cli/ProjectCommand.java

```diff
@@ -307,18 +308,31 @@ public boolean isExperimentalIntelliJProjectGenerationEnabled() {
     return experimentalIntelliJProjectGenerationEnabled;
   }
 
-  public List<String> getArgumentsFormattedAsBuildTargets(BuckConfig buckConfig) {
-    return getCommandLineBuildTargetNormalizer(buckConfig).normalizeAll(getArguments());
-  }
-
   @Override
   public int runWithoutHelp(CommandRunnerParams params) throws IOException, InterruptedException {
     if (getIde(params.getBuckConfig()) == ProjectCommand.Ide.XCODE) {
       checkForAndKillXcodeIfRunning(params, getIdePrompt(params.getBuckConfig()));
     }
 
-    ImmutableSet<BuildTarget> passedInTargetsSet = getBuildTargets(
-        getArgumentsFormattedAsBuildTargets(params.getBuckConfig()));
+    Pair<ImmutableSet<BuildTarget>, TargetGraph> result = null;
```

Comment:
@Coneko left this comment on our internal tracker but forgot to put it here so you can address it.

The TargetGraph from here should be used to generate the project as well instead of creating a new one using the targets below.